src/Octoshift/Services/ReclaimService.cs

```diff
@@ -151,15 +149,21 @@ public virtual async Task ReclaimMannequins(string[] lines, string githubTargetO
 
             if (!force && mannequins.IsClaimed(login, userid))
             {
-                _log.LogError($"{login} is already claimed. Skipping (use force if you want to reclaim)");
+                _log.LogWarning($"{login} is already claimed. Skipping (use force if you want to reclaim)");
                 continue;
             }
 
             var mannequin = mannequins.FindFirst(login, userid);
 
             if (mannequin == null)
             {
-                _log.LogError($"Mannequin {login} not found. Skipping.");
+                _log.LogWarning($"Mannequin {login} not found. Skipping.");
+                continue;
+            }
+
+            if (lines.Where(x => x.Contains(login)).Count() > 1 && lines.Where(x => x.Contains(userid)).Count() > 1)
```

Comment:
This appears to be looking at the entire unparsed CSV line without looking at the individual column values (and the commas, too).  Can we inspect the parsed CSV values instead?

Reply:
Broke out parsing and validation here: 9dfd63e535ffb35b4434178b2ab6c7a7a89e1a94